frontend/auth/configProfileController.js

```diff
@@ -47,8 +47,8 @@
             });
         }
 
-        configProfileCtrl.cropImage = function cropImage(imageFile) {
-            CropImageService.crop(imageFile).then(function success(croppedImage) {
+        configProfileCtrl.cropImage = function cropImage(imageFile, event) {
```

Comment:
Mesmo tratamento deve ser feito ao trocar a imagem da instuição em ConfigInstController

Reply:
Fix then it's ok